rel-eng/atomic-enterprise-master.service

```diff
@@ -3,16 +3,14 @@ Description=Atomic Enterprise Master
 Documentation=https://github.com/openshift/origin
 After=network.target
 After=etcd.service
-Before=atomic-enterpries-node.service
-Before=atomic-enterprise-sdn-node.service
-Before=atomic-enterprise-sdn-master.service
+Before=atomic-enterprise-node.service
 Requires=network.target
 
 [Service]
 Type=notify
-EnvironmentFile=/etc/sysconfig/atomic-enterprise-master
+EnvironmentFile=/etc/sysconfig/origin-master
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/atomic-enterprise start master --config=${CONFIG_FILE} $OPTIONS
+ExecStart=/usr/bin/openshift start master --config=${CONFIG_FILE} $OPTIONS
```

Comment:
We don't actually have `/usr/bin/origin` but I can see how it's confusing having the services named `atomic-enterprise-master` but running `openshift` binary.

@brenton I'm not sure what the desired binaries that are invoked via the services are.